platform/osgi/platform-osgi-configadmin/src/test/java/org/codice/felix/cm/file/EncryptingPersistenceManagerTest.java

```diff
@@ -64,16 +65,30 @@
 
   @Mock private PersistenceManager mockManager;
 
-  private EncryptingPersistenceManager persistenceManager;
+  private String pwDirectory;
 
   @Before
   public void setup() throws Exception {
-    String pwDirectory = temporaryFolder.newFolder().getAbsolutePath();
-    persistenceManager = new EncryptingPersistenceManager(mockManager, pwDirectory);
+    pwDirectory = temporaryFolder.newFolder().getAbsolutePath();
+  }
+
+  @Test
```

Comment:
📘 This test tests if mutliple instances of this class will use the same `keyset`.